install_ddev.sh

```diff
@@ -46,46 +46,50 @@ ${GREEN}
 ${RESET}"
 while true; do
     read -p "Continue? (y/n): " INSTALL
-    case $INSTALL in
+    case "$INSTALL" in
         [Yy]* ) break;;
         [Nn]* ) exit;;
         * ) echo "Please answer y or n.";;
     esac
 done
 
-if ! docker run -t busybox:latest ls >/dev/null; then
-    printf "
-    ${RED}
-    ####
-    # You need to install Docker and have it running before executing this script.
-    # The installer may be provided with this package.
-    # Docker installation and troubleshooting information is at
-    # https://ddev.readthedocs.io/en/latest/users/docker_installation/
-    ####
-    ${RESET}"
-    exit 1
-fi
-
 echo ""
 echo "Installing docker images for ddev to use..."
-if [[ "$OS" == "Darwin" ]]; then
-    gzip -dc $(ls ddev_tarballs/ddev_docker_images*.tar.xz) | docker load
-elif [[ "$OS" == "Linux" ]]; then
-    xzcat $(ls ddev_tarballs/ddev_docker_images*.tar.xz) | docker load
-fi
+7z x ddev_tarballs/ddev_docker_images.*.tar.xz -so | docker load
```

Comment:
I'm okay with removing the commands for Windows users and relying on Git Bash or Docker Quickstart Terminal.

A year ago I ran into issues with trying to get 7zip to run in those environments on my own docker-core-sprint, but it's possible that Git Bash (with or without Docker Quickstart Terminal) did not include it at that point (?). Need to test this on Docker Toolbox.

Reply:
I think that Powershell is probably preferable for Window 10 Pro users, but I don't think we can support that because of the numerous Windows 10 versions and whatever they support.